### PR TITLE
🏃 Add release targets

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -150,6 +149,33 @@ docker-push-manifest: ## Push the fat manifest docker image.
...
 	@for arch in $(ALL_ARCH); do docker manifest annotate --arch $${arch} ${CONTROLLER_IMG}:${TAG} ${CONTROLLER_IMG}-$${arch}:${TAG}; done
-	docker manifest push --purge ${CONTROLLER_IMG}:${TAG}
-	@echo "updating kustomize image patch file for manager resource"
-	sed -i'' -e 's@image: .*@image: '"${CONTROLLER_IMG}:$(TAG)"'@' ./config/default/manager_image_patch.yaml
+	MANIFEST_IMG=$(CONTROLLER_IMG) MANIFEST_TAG=$(TAG) $(MAKE) set-manifest-image
+
+.PHONY: set-manifest-image
+set-manifest-image:
+	$(info Updating kustomize image patch file for manager resource)
+	sed -i'' -e 's@image: .*@image: '"${MANIFEST_IMG}:$(MANIFEST_TAG)"'@' ./config/default/manager_image_patch.yaml
```
[the remark this answers is not in the document]
Comment (chuckha) [reply Makefile]: although you can just add a comment if you want this to only run via action and not run by hand

[the remark this answers is not in the document]
Comment (vincepri) [reply Makefile]: This is already in the .gitignore `manager_image_patch.yaml-e`

[the remark this answers is not in the document]
Comment (chuckha) [reply Makefile]: yeah, i think that got added before we realized the generated default sed command was for gnu sed

[the remark this answers is not in the document]
Comment (vincepri) [reply Makefile]: Fun times

**Pull request merged**